### PR TITLE
Expand testing section a bit

### PR DESCRIPTION
**Pull request by standage**

### CODE REVIEW

File: short/index.html [hunk 1]
```diff
@@ -179,16 +179,30 @@
...
-    *   Unit tests (developer-facing)                     <!-- testing at scale of individual functions/methods -->
-    *   Functional tests (user-facing)                    <!-- testing at scale of entire reusable components (scripts, classes, libraries) -->
-    *   Regression tests (bug-facing)                     <!-- written when a bug is fixed -->
-*   Execute frequently to ensure reliability of software  <!-- setting up for discussion of continuous integration -->
+*   Execute tests automatically and frequently
+    *   Ensures code changes don't break what already works <!-- setting up for discussion of continuous integration -->
+</section>
+
+<section data-markdown>
+**Automated Testing (cont.)**
```
Comment (gvwilson) [inline short/index.html:191]: Change title to "Kinds of Tests"

File: short/index.html [hunk 2]
```diff
@@ -179,16 +179,30 @@
...
+
+*   Smoke tests (sanity checks)     <!-- literally, make sure the code doesn't blow up when invoked -->
+*   Unit tests (developer-facing)   <!-- testing at scale of individual functions/methods -->
+*   Functional tests (user-facing)  <!-- testing at scale of entire reusable components (scripts, classes, libraries) -->
+*   Regression tests (bug-facing)   <!-- written when a bug is fixed -->
+</section>
+
+<section data-markdown>
```
Comment (gvwilson) [inline short/index.html]: Move this one up above "Kinds of Tests"

File: short/index.html (hunk 1 above)
Comment (standage) [reply short/index.html:191]: Done.

File: short/index.html (hunk 2 above)
Comment (standage) [reply short/index.html]: Done.

**Pull request merged**